build.gradle

```diff
@@ -14,16 +20,16 @@ ext {
         moduleVersion = System.getProperty('moduleVersion')
     }
 
-    group 'eu.tsystems.mms.tic.testerra'
+    group 'io.testerra'
     version moduleVersion
 }
 
 // Do not move this integration because `group` and `version` is needed for publishing
 apply from: rootProject.file('publish.gradle')
 
 dependencies {
-    compileOnly 'eu.tsystems.mms.tic.testerra:core:' + testerraCompileVersion
-    implementation 'eu.tsystems.mms.tic.testerra:surefire-connector:' + testerraCompileVersion
+    compileOnly 'io.testerra:core:' + testerraCompileVersion
+    implementation 'io.testerra:surefire-connector:' + testerraCompileVersion
```

Comment:
Should we make all dependencies `compileOnly`, what you think?

Reply:
Yes, thats correct.